docs/source/commandline/commandline-all.rst

```diff
@@ -928,95 +1148,176 @@ Core Video Options
+    size (even vertical systems).
 
-	The MAME team heavily suggests you leave this at default. Stretching systems beyond their original aspect ratio will mangle the appearance of the system in ways that no filtering or HLSL can repair.
+    In window mode, it means that you can freely resize the window without any
+    constraints.
+
+    The default is ON (**-keepaspect**).
+
+    The MAME team heavily suggests you leave this at default. Stretching systems
+    beyond their original aspect ratio will mangle the appearance of the system
+    in ways that no filtering or HLSL can repair.
 
 .. _mame-commandline-waitvsync:
 
 **-[no]waitvsync**
 
-	Waits for the refresh period on your computer's monitor to finish before starting to draw video to your screen. If this option is off, MAME will just draw to the screen at any old time, even in the middle of a refresh cycle. This can cause "tearing" artifacts, where the top portion of the screen is out of sync with the bottom portion. Tearing is not noticeable on all systems, and some people hate it more than others. However, if you turn this option on, you will waste more of your CPU cycles waiting for the proper time to draw, so you will see a performance hit. You should only need to turn this on in windowed mode. In full screen mode, it is only needed if **-triplebuffer** does not remove the tearing, in which case you should use **-notriplebuffer -waitvsync**. Note that this option does not work with **-video gdi** mode. The default is OFF (**-nowaitvsync**).
+    Waits for the refresh period on your computer's monitor to finish before
+    starting to draw video to your screen.  If this option is off, MAME will
+    just draw to the screen at any old time, even in the middle of a refresh
+    cycle.  This can cause "tearing" artifacts, where the top portion of the
+    screen is out of sync with the bottom portion.  Tearing is not noticeable
+    on all systems, and some people hate it more than others.  However, if you
+    turn this option on, you will waste more of your CPU cycles waiting for the
+    proper time to draw, so you will see a performance hit.
+
+    You should only need to turn this on in windowed mode. In full screen mode,
+    it is only needed if **-triplebuffer** does not remove the tearing, in which
+    case you should use **-notriplebuffer -waitvsync**. Note that this option
+    does not work with **-video gdi** mode.
+
+    The default is OFF (**-nowaitvsync**).
 
-	Note that SDL-based MAME support for this option depends entirely on your operating system and video drivers; in general it will not work in windowed mode so **-video opengl** and fullscreen give the greatest chance of success.
+    Note that SDL-based MAME support for this option depends entirely on your
+    operating system and video drivers; in general it will not work in windowed
+    mode so **-video opengl** and fullscreen give the greatest chance of
+    success.
```

Comment:
This option's behaviour is depends on your operating system and video drivers.  On Windows it blocks until vblank before allowing the application to draw a frame, so it will limit the frame rate to the host screen refresh rate.  On macOS it's effectively transparent buffering - the application will not be slowed down, but the most recent complete frame will be drawn at vblank so if the emulated system has a higher refresh rate than the host, frames will be dropped causing motion judder.

Reply:
We'll probably end up talking about my proposed changes on this one. I'm not expecting this one to be acceptable on the second pass.